src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
```

Comment:
The message should indicate more specifically that it is an `Excavator` that is working. For example, 'Excavator is working.' would be more informative.